modules/rules_gazebo/0.0.1/MODULE.bazel

```diff
@@ -0,0 +1,12 @@
+"rules_gazebo"
+
+module(
+    name = "rules_gazebo",
+    version = "0.0.1",
+    compatibility_level = 1,
+)
+
```

Comment:
Needs a `bazel_dep` for `rules_license`.

Reply:
Done in version 0.0.2